src/gtest/test_joinsplit.cpp

```diff
@@ -547,3 +547,25 @@ TEST(joinsplit, note_plaintexts)
 
     ASSERT_TRUE(decrypted.memo == note_pt.memo);
 }
+
+TEST(joinsplit, note_class)
+{
+    uint252 a_sk = uint252(uint256S("f6da8716682d600f74fc16bd0187faad6a26b4aa4c24d5c055b216d94516840e"));
```

Comment:
Is this intended to be a specific value, or random? If the latter, use `uint252 a_sk = uint252(random_uint256());` instead.